### PR TITLE
Remove store for rootSection + add request helper

### PR DESCRIPTION
**Pull request by Matsa59**
The request helper could be use to "resolve" `_links`.

### CODE REVIEW

**Review by onixus74 — APPROVED** [review]
Sweet 👍

File: README.md [hunk 1]
```diff
@@ -20,40 +20,29 @@ npm i @omerlo/omerlo-webkit
...
-};
-```
-
-Add the `omerloWebkit` plugin to `vite.config.ts`
+- `OMERLO_PUBLIC_BASE_API_URL`: base URL of the public api.
+- `OMERLO_PUBLIC_MEDIA_ID`: the media you want to display in your website.
 
 ```js
-// vite.config.ts
+// vite.config.ts or vite.config.js
```
Comment (PatrickD26) [inline README.md:29]: Do we want to include the possibility of not using TS ? Maybe it's better to enforce a behaviour and have a  clean workflow accross the different projects

**Review by PatrickD26 — APPROVED** [review]
LGTM, see comment

File: README.md (hunk 1 above)
Comment (Matsa59) [reply README.md:29]: I can't / don't want to enforce people to use TS over JS :/

**Review by NatoBoram — APPROVED** [review]

**Pull request merged**